packages/template/package.json

```
@@ -28,8 +28,8 @@
     "test-snippets": "yarn extract-snippets && yarn g:tsc -p codeSnippets/tsconfig.build.json"
   },
   "dependencies": {
-    "@cognite/sdk": "^9.17.1",
-    "@cognite/sdk-core": "^4.10.7"
+    "@cognite/sdk": "workspace:*",
+    "@cognite/sdk-core": "workspace:*"
```

```suggestion
    "@cognite/sdk": "^10.4.0",
    "@cognite/sdk-core": "^5.1.3"
```
it's a template package, it should be aligned with the other packages in the folder.
If you look to a specific package (e.g. beta) it locks the version more strictly

---

changed version